docs-win/index.md

```diff
@@ -0,0 +1,297 @@
+## Overview
+
+ezbeq is a web interface that works in conjunction with minidsp-rs, which allows for simplistic selection of a movie / TV show BEQ filter and configuration of a MiniDSP 2x4HD, without having to use the proprietary minidsp plugin. The web interface is usable from phones, tablets, laptops or desktops.
+
+## Pre-Requirements
+
+ezbeq and minidsp-rs run on Linux, Windows, or Mac operating systems, but this document walks through the installation process on Windows 10. The following systems are required to follow this document:
+
+- MiniDSP 2x4HD
+- Windows 10 system (this document based on 20H2, fully patched)
+- USB cable connecting the Windows device and MiniDSP 2x4HD
+- Internet connection
+- Backup copy of all MiniDSP 2x4HD settings
+
+NOTE – ezbeq and minidsp-rs will be modifying the INPUT settings of the MiniDSP 2x4HD, but please take appropriate backups.
+
+## Installation Steps
+
+### Install Microsoft Visual Studio Build Tools
+Both minidsp-rs and one of the pre-requisites for ezbeq requires the Build Tools to be available, or installation and execution will fail.
```

Comment:
FWIW there is an exe published on windows so it can be used without these steps

Reply:
The minidsp-rs win exe requires VC runtime. It either has to be downloaded separately, or it is installed as part of the VS Build Tools install (needed for cryptography), so that just did it in one step.